src/components/Button/Button.module.css

```diff
@@ -8,12 +8,14 @@
  * 1) Button or link that has functionality to it
  * 2) Defaults to primary button
  * 3) Margin 0 added becuase Safari has margin on the button
+ * 4) Adds a gap between children to automatically handle spacing between icons and text
  */
 .button {
   display: inline-flex;
   align-items: center;
   justify-content: center;
   margin: 0; /* 3 */
+  gap: 0.75rem; /* 4 */
```

Comment:
If you look at the deleted code below, this number varied depending on the button variant and size, but it's always the same in our mocks. (Also the font size no longer changes at different button sizes, which may have been why the gap shrank with smaller buttons.)